db/migrate/20241206155422_create_addresses.rb

```diff
@@ -0,0 +1,13 @@
+class CreateAddresses < ActiveRecord::Migration[7.2]
+  def change
+    create_table :addresses do |t|
+      t.string :address, null: false
+      t.string :city, null: false
+      t.string :state, null: false
+      t.string :postal_code, null: false
+      t.references :addressable, null: false, foreign_key: true
```

Comment:
Did this migration run? I'm surprised since `Addressable` is not a model.

Reply:
it did. 
then I deleted foreign_key: true, and added polymorphic: true instead in the next migration